language_server/src/dap/dap_feature.cpp

```diff
@@ -148,8 +149,8 @@ void dap_feature::on_launch(const request_id& request_seq, const nlohmann::json&
         return;
 
     // wait for configurationDone?
-    std::string program_path = server_conformant_path(args["program"].get<std::string_view>(), client_path_format_);
-    bool stop_on_entry = args["stopOnEntry"].get<bool>();
+    std::string program_path = server_conformant_path(args.at("program").get<std::string_view>(), client_path_format_);
```

Comment:
To unify with the change below:
```suggestion
    auto program_path = server_conformant_path(args.at("program").get<std::string_view>(), client_path_format_);
```